iac-extensions/terraform/src/main/java/org/sonar/iac/terraform/checks/DisabledLoggingCheck.java

```diff
@@ -161,4 +162,19 @@ private static void checkGlobalAccelerator(CheckContext ctx, BlockTree resource)
           reportOnFalse(ctx, enabled, MESSAGE), () -> ctx.reportIssue(attributes.key(), MESSAGE)),
       () -> reportResource(ctx, resource, MESSAGE));
   }
+
+  private static void checkElasticSearchDomain(CheckContext ctx, BlockTree resource) {
+    PropertyUtils.getAll(resource, "log_publishing_options", BlockTree.class).stream()
+      .filter(DisabledLoggingCheck::isAuditLog)
+      .findAny()
```

Comment:
Let's use `findFirst()` to make the order relevance more explicit, https://stackoverflow.com/a/35359193 .